packages/form/src/Bind.ts

```diff
@@ -48,7 +48,6 @@ const createBind = (form: Form) => {
 
         // Build new input props
         const newProps = {
-            disabled: false,
```

Comment:
This will cause TS error on line 70:
![image](https://user-images.githubusercontent.com/5121148/77847098-c6b7a000-71ba-11ea-9cd7-265891691cc6.png)

I would maybe undo this change,  and add the following piece on line 76:
```js
if (!newProps.disabled) {
   newProps.disabled = children.props.disabled;
}
```

Basically, if the `disabled={true}` prop wasn't present on the whole `Form` component, then we should just take the `disabled` value from the actual child component, which the `Bind` component is wrapping.

Reply:
yes, those error also happened in my local machine. ok, so am i just have to revert the change?

Reply:
Yeah, revert it, add the `if` statement mentioned above, and we should be good to go I believe.

Reply:
unfortunately, i have tried that and it says typescript error, because there's no props property on children 🤔 

![Screen Shot 2020-03-29 at 19 44 12](https://user-images.githubusercontent.com/52902060/77849417-ace77e80-71f5-11ea-9e3d-4ea0e617cd4c.png)

Reply:
This needs to go inside of the `if (React.isValidElement(children)) { ...` statement.

Reply:
ok thanks for pointing out @doitadrian